kibana.yaml

```diff
@@ -30,3 +30,31 @@ spec:
         ports:
         - containerPort: 5601
           name: http
+        livenessProbe:
+          failureThreshold: 1
+          httpGet:
+            path: /api/status
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 2
+          successThreshold: 1
+          timeoutSeconds: 3
+        readinessProbe:
+          failureThreshold: 1
+          httpGet:
+            path: /api/status
+            port: http
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 2
+          successThreshold: 1
+          timeoutSeconds: 3
```

Comment:
What's the purpose of having liveness and readiness probes that are exactly the same?

Reply:
Also, `failureThreshold`, `periodSeconds` and `successThreshold` seem quite agggressive to me. Why these values?

Reply:
I've encounter a case that Kibana not responding.
that's why I added the `readinessProbe` here.

Reply:
If the liveness and readiness probes are the same, as soon as liveness passes, so will readiness. It seems redundant to me.